tensorbay/client/requests.py

```diff
@@ -135,7 +135,7 @@ def __init__(self) -> None:
 
     def request(  # type: ignore[override]  # pylint: disable=signature-differs
         self, method: str, url: str, *args: Any, **kwargs: Any
-    ) -> Response:
+    ) -> Response:  # noqa: DAR401
```

Comment:
Why disable `DAR401`?

Reply:
Flake8 only recognizes the literal word behind `raises` instead of the type of the following exception.
So it considers the exception raised by L162 `raise ResponseErrorDistributor[error_code](response)` not included in the docstring.